docs/source/fitting/fitfunctions/AsymmetricPearsonVII.rst

```diff
@@ -0,0 +1,81 @@
+.. _func-AsymmetricPearsonVII:
+
+======================
+Asymmetric Pearson VII
+======================
+
+.. index:: AsymmetricPearsonVII
+
+Description
+-----------
+
+The asymmetric Pearson VII function (sometimes it is also referred to as the split-Pearson VII function) is a function that combines
+two Pearson VII distributions so to fit sharp peak curves. It is useful for analysis of X-ray diffraction peaks which consist of two separable components.
+
+In one of the representations, the asymmetric Pearson VII function :math:`A(x)` can be written as a superposition of the "left" and "right" Pearson VII functions :math:`P_L(x)` and :math:`P_R(x)`,
+
+.. math:: A(x) = P_L(x) \, \theta(x_0 - x) + P_R(x) \, \theta(x - x_0),
+
+where :math:`x_0` denotes the coordinate of the peak centre, :math:`\theta` denotes the Heavyside step function with :math:`\theta(y \ge 0) = 1` and :math:`\theta(y < 0)=0`, and :math:`P(x)` denotes the Pearson VII function. This function takes four additional parameters and in general is defined as
+
+.. math:: P(x, h, x_0, \Gamma, m) = \frac{h}{\left(1 + \left[ \frac{2 (x-x_0) \sqrt{2^{1/m}-1}}{\Gamma} \right]^2 \right)^m} \, .
```

Comment:
I cannot check this equation without a reference. I have had a quick look online but they seems to have different assumptions (some terms set to 1). 

Also the square root seems odd given that it is later squared. Would it not be simpler to write:

\frac{4(x-x_0)^2 (2^{-m}-1)}{Gamma^2}

Reply:
comparing with the equation in http://pd.chem.ucl.ac.uk/pdnn/peaks/pvii.htm am I right in thinking:

h = w^{2m}
x/Gamma = theta
x_0/Gamma = theta_0
w^2 seems to be one, which is inconsistent with the definition of h, unless h is also 1

Have I misunderstood something or is this the wrong equation?

Reply:
What is the benefit of the rearrangement? Does it make the variables have more direct physical counterparts?

Reply:
I have seen various forms of this expression in the literature and it's hard to judge why someone prefers to use one form over another. I'd say, this is more a matter of taste to operate with a full-width at half maximum or with a half-width at half-maximum. As regarding to keeping Gamma^{2m} in the numerator, there might be a numerical benefit in doing that - one can avoid 1/0 singularities as Gamma-->0.